helm/prometheus-rules/templates/alerting-rules/kvm.management-cluster.rules.yml

```diff
@@ -12,6 +12,28 @@ spec:
   groups:
   - name: kvm
     rules:
+    - alert: BlackboxExporterEndpointDown
+      annotations:
+        description: '{{`Blackbox exporter endpoint ({{ $labels.instance }}) is down.`}}'
+        opsrecipe: blackbox-exporter-endpoint-down/
+      expr: up{job="management-cluster-blackbox-ingress-http"} == 0
+      for: 15m
+      labels:
+        area: kaas
+        severity: page
+        team: rocket
+        topic: observability
+    - alert: BlackboxExporterEndpointNotResponding
+      annotations:
+        description: '{{`Blackbox exporter endpoint ({{ $labels.instance }}) is not responding.`}}'
+        opsrecipe: blackbox-exporter-endpoint-down/
+      expr: sum by(instance) (avg_over_time(probe_http_duration_seconds[5m])) > 0.2
```

Comment:
Out of curiosity, why 0.2 secs?

Reply:
It was chosen arbitrarily based on the last few days of metrics. I wanted it to be high enough that normal variation didn't cause it to fire, but real issues would. I figure we can adjust it downwards if we miss a problem we should have been paged for or upwards if we're getting spurious alerts.